bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoVacuumHandler.java

```diff
@@ -131,16 +131,18 @@ private boolean updateVacuumStatus(JsonObject statusData) {
         updateState(CHANNEL_CLEAN_TIME,
                 new DecimalType(TimeUnit.SECONDS.toMinutes(statusData.get("clean_time").getAsLong())));
         updateState(CHANNEL_DND_ENABLED, new DecimalType(statusData.get("dnd_enabled").getAsBigDecimal()));
-        updateState(CHANNEL_ERROR_CODE,
-                new StringType(VacuumErrorType.getType(statusData.get("error_code").getAsInt()).getDescription()));
+        VacuumErrorType errorCode = VacuumErrorType.getType(statusData.get("error_code").getAsInt());
```

Comment:
What is the benefit of writing this in two lines instead of one?

Reply:
This has no real advantage. I had only taken it from the original code to make it consistent.